Mathlib/NumberTheory/Padics/ProperSpace.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jou Glasheen, Kevin Buzzard
 -/
 
-import Mathlib.Analysis.Normed.Module.FiniteDimension
+import Mathlib.Analysis.Normed.Field.ProperSpace
 import Mathlib.NumberTheory.Padics.RingHoms
 
```

Comment:
And it would be excellent to add at least one `assert_not_exists` statement here, based on what you've just removed.